csharp_dotnetcore/51.Enterprise-Bot-Template/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "microsoftAppId": "",
```

Comment:
This needs to have the botFilePath and botFileSecret

Reply:
Added.